src/commands/edge_app_server.rs

```diff
@@ -110,42 +110,7 @@ impl Default for Metadata {
 #[derive(Debug, Default, Deserialize)]
 struct MockData {
     metadata: Metadata,
-    settings: Vec<(String, Value)>,
-}
-
-#[derive(Debug, Default, Deserialize, Clone)]
-struct MockDataStr {
-    metadata: Metadata,
-    settings: BTreeMap<String, String>,
-}
-
-impl MockData {
-    fn new_from_str(mock_data_str: &MockDataStr) -> Self {
-        let settings_val = mock_data_str
-            .settings
-            .iter()
-            .map(|(k, v)| {
-                if v.len() > 2
-                    && v.chars().nth(0) == Some('[')
-                    && v.chars().nth(v.len() - 1) == Some(']')
-                {
-                    let v = &v[1..(v.len() - 1)];
-                    let v = v
-                        .split(',')
-                        .map(|s| s.trim().replace(['"', '\''], ""))
```

Comment:
What happens if the value includes quotes? This custom deserializer was also discarding them.

Reply:
It's a proper yaml parser. It won't bring in quotes denoting the string.

Reply:
Our test `test_server_should_serve_correct_mock_data` already tests exactly for that:

![image](https://github.com/Screenly/cli/assets/916132/18dc02c5-55bd-4cbc-a1b4-3358f09d0bb2)

Reply:
Also does not that mean your code would not handle " inside the string? It's valid to have it for strings.

Reply:
If that's taken care of, I'm OK. It was an issue in the beginning I guess but I might be recalling it incorrectly.